docs/addons/webhook.md

```diff
@@ -0,0 +1,8 @@
+---
+id: webhook
+title: Unleash Webhook addon
+---
+
+> This feature was introduced in _Unleash v3.11.0_.
+
+Unleash Addons allows you to extend Unleash with new functionaluty
```

Comment:
```suggestion
Unleash Addons allows you to extend Unleash with new functionality
```
Though - the correct change here is probably to document the webhook addon, rather than copypasta from the generic addon documentation :smile:

Reply:
Which reminds me that we should document the other addons as well.